src/Notifications/onlineTemplates.js

```diff
@@ -6,72 +6,84 @@ async function generateOnlineTemplate(template, to, from) {
             submitter: {
                 toUser: from.gcID,
                 fromUser: from.gcID,
-                email: from.email,
                 titleEn: "Team Transfer",
                 titleFr: "Transfert d'équipe",
                 descriptionEn: `Your request to transfer a team to ${to.name} has been created`,
                 descriptionFr: `Votre demande de transferer une equipe à ${to.name} a été créée`,
+                email: from.email,
                 actionLevel: "NoUserAction",
                 emailSubject: `Team Transfer | Transfert d'équipe`,
+                emailBodyEn: `Your request to transfer a team to ${to.name} has been created`,
+                emailBodyFr: `Votre demande de transferer une equipe à ${to.name} a été créée`,
             },
             approver: {
                 toUser: to.gcID,
                 fromUser: from.gcID,
-                email: to.email,
                 titleEn: `Team Transfer from ${from.name}`,
                 titleFr: `Demande de transfert d'équipe de ${from.name}`,
                 descriptionEn: `${from.name} is requesting to transfer you a team.  Please approve or deny this request`,
                 descriptionFr: `${from.name} demande de vous transférer une équipe. Veuillez approuver ou refuser cette demande`,
                 actionLevel: "UserActionRequired",
+                email: to.email,
                 emailSubject: `Team Transfer from ${from.name} | Demande de transfert d'équipe de ${from.name}`,
+                emailBodyEn: `${from.name} is requesting to transfer a team to you.  Please approve or deny this request.`,
+                emailBodyFr: `${from.name} demande de vous transférer une équipe. Veuillez approuver ou refuser cette demande.`,
```

Comment:
In english you add the "Please" but not in french.  On line 86, for the same sentence, you add the "s.v.p" in french.